<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="BoxOffice">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/new/models/BoxOffice.svg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="BoxOffice_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>240</y>
      <width>261</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/new/models/BoxOffice.svg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="men1">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>141</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/new/models/men-1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="men1_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>250</y>
      <width>141</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/new/models/men-1.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="PeopleCount1">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color : white; }</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="PeopleCount2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>340</y>
      <width>41</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel { color : white; }</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cashe1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="cashe2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>430</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>291</width>
      <height>291</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="res.qrc">:/new/models/church-filled.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QDial" name="clock">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>50</width>
      <height>64</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>* 10 пасс/час</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>154</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>secPerMin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="secPerMinute"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>clock</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>824</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStart">
    <property name="title">
     <string>Action</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionReStart"/>
   </widget>
   <widget class="QMenu" name="menuStatistic">
    <property name="title">
     <string>Statistic</string>
    </property>
    <addaction name="actionStatistic"/>
   </widget>
   <addaction name="menuStart"/>
   <addaction name="menuStatistic"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionReStart">
   <property name="text">
    <string>ReStart</string>
   </property>
  </action>
  <action name="actionStatistic">
   <property name="text">
    <string>Statistic</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clock</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>611</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>745</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
